examples/aggregator/proto/aggregator.proto

```diff
@@ -0,0 +1,35 @@
+//
+// Copyright 2020 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+syntax = "proto3";
+
+package oak.examples.aggregator;
+
+import "google/protobuf/empty.proto";
+
+message SubmitSampleRequest {
+  repeated uint64 values = 1;
```

Comment:
I think it would be simpler to define a separate type, e.g.

```protobuf
message Vector {
  repeated uint64 values = 1;
}
```

and reuse that in both request and response. Also I think it makes the rest of the code more idiomatic, you can declare `impl Monoid for Vector` directly.

Reply:
Will it be ok to return an empty `Vector` if there are not enough samples?
Because if I return an error from the gRPC method - then the application stops working.

Reply:
I think it's fine to have a Response wrapper message, with a single field of type `Vector`, in which case you can leave that unset.

But in any case, why does it stop working? Is there an issue opened for it already? If not, could you create it (and perhaps look into it?)

Reply:
I'll create an issue. Just thought that it was a normal behavior at first.

Reply:
Also, by reusing `Vector` in both request and response - do you mean we will still need `SubmitSampleRequest` messages, or just use:
```protobuf
rpc SubmitSample(Vector) returns (google.protobuf.Empty);
```

Reply:
Up to you, perhaps keep both Request and Response messages for now.

Reply:
Done